### PR TITLE
feat: add responsive adjustments on account screen

### PR DESCRIPTION
**Pull request by pedronauck**
⭐ Closes [FE-4](https://linear.app/fuel-network/issue/FE-4/account-page-mobile-ui)

closes FE-4

### CODE REVIEW

**Review by LuizAsFight — CHANGES_REQUESTED** [review]
not enough space to show balance value

image

File: packages/ui/src/hooks/useBreakpoints.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { useMedia } from 'react-use';
+
+export function useBreakpoints() {
+  const isMobile = useMedia('(max-width: 576px)');
+  const isTablet = useMedia('(min-width: 577px)');
```
Comment (LuizAsFight) [inline packages/ui/src/hooks/useBreakpoints.ts:5]: value here is wrong right? tablet.is `640px`

**Review by LuizAsFight — CHANGES_REQUESTED** [review]

**Review by LuizAsFight — CHANGES_REQUESTED** [review]
tabs should be horizontal also when in mobile. 

 like in the mockups

image

**Comment by pedronauck** [comment]
> tabs should be horizontal also when in mobile.
> 
> like in the mockups
> 
> image

@LuizAsFight  ✅

File: packages/ui/src/hooks/useBreakpoints.ts (hunk 1 above)
Comment (pedronauck) [reply packages/ui/src/hooks/useBreakpoints.ts]: I tried to use the same value, the only way it's able is being using javascript variables, by using css vars wasn't possible

**Comment by pedronauck** [comment]
> not enough space to show balance value
> 
> image

@LuizAsFight I don't know about why you were talking here, but I guess it was fixed by some merge

**Review by LuizAsFight — CHANGES_REQUESTED** [review]
image

you don't need this padding in the internal content, padding is handled by rootlayout now

File: packages/app/src/app/account/[id]/transactions/page.tsx [hunk 2]
```diff
@@ -11,7 +11,13 @@ export default async function AccountTransactions({
   params: { id },
 }: PageProps) {
   const txs = await getAccountTransactions({ owner: id });
-  return <TxList hidePagination transactions={txs.edges} />;
+  return (
+    <TxList
+      hidePagination
+      transactions={txs.edges}
+      className="mobile:p-0 tablet:p-0 desktop:p-0"
```
Comment (LuizAsFight) [inline packages/app/src/app/account/[id]/transactions/page.tsx:18]: `mobile` is not needed , we can use as default one

this applies also for other usages of `mobile:` in this PR

```suggestion
      className="p-0 tablet:p-0 desktop:p-0"
```

File: packages/ui/src/theme/tailwind-preset.ts [hunk 3]
```diff
@@ -87,9 +88,10 @@ const preset: Config = {
         ...refColorVariablesAsObj(),
       },
       screens: {
-        tablet: '640px',
-        laptop: '960px',
-        desktop: '1280px',
+        mobile: `${breakpoints.mobile}px`,
```
Comment (LuizAsFight) [inline packages/ui/src/theme/tailwind-preset.ts:91]: mobile is not needed

**Review by LuizAsFight — CHANGES_REQUESTED** [review]

**Review by LuizAsFight — APPROVED** [review]

**Pull request merged**